schema/swagger.yml

```diff
@@ -595,6 +595,46 @@ paths:
           schema:
             $ref: '#/definitions/UnexpectedError'
 
+  /delegates/{publicKey}/forged:
```

Comment:
I believe this `forged` term conceived by old API. But what if we use `forging_stats` or just `/delegates/{}/stats` that makes more meaningful endpoint.